bench/query.go

```diff
@@ -132,6 +132,90 @@ func (q *QueryGenerator) Random(maxN, depth, maxargs int, idmin, idmax uint64) *
 	}
 }
 
+// RandomRangeQuery returns a randomly generated either sum/range query.
+func (q *QueryGenerator) RandomRangeQuery(depth, maxargs int, frame, field string, idmin, idmax uint64) *pql.Call {
+	val := q.R.Intn(5)
+	switch val {
```

Comment:
probably don't need `val` here. you can just put `q.R.Intn(5)` in the switch statement.